content/en/docs/concepts/overview/components.md

```diff
@@ -50,79 +81,74 @@ the same machine, and do not run user containers on this machine. See
 
 These controllers include:
 
-  * Node Controller: Responsible for noticing and responding when nodes go down.
-  * Replication Controller: Responsible for maintaining the correct number of pods for every replication
-  controller object in the system.
-  * Endpoints Controller: Populates the Endpoints object (that is, joins Services & Pods).
-  * Service Account & Token Controllers: Create default accounts and API access tokens for new namespaces.
+  * Deployment controller: Responsible for maintaining the correct number of pods for
```

Comment:
Technically, ReplicaSet maintains the correct number of pods. Deployment is more for ensuring the pods run with up-to-date configurations. For simplicity, the two could be combined:

Responsible for maintaining the correct numbers of pods for stateless applications, and ensuring they are running with up-to-date configurations.